pandas/io/parsers.py

```diff
@@ -73,6 +73,50 @@
 
     By file-like object, we refer to objects with a ``read()`` method, such as
     a file handler (e.g. via builtin ``open`` function) or ``StringIO``.
+
+    Occasionally, you might run into a data file that begins with comments.
+
+    .. code-block:: text
+
+        # data.csv
+        # This file has info on Apollo space missions.
+        # Each row has info on the mission title, launch date, and duration.
+        duration,launch_date,mission
+        NaN,21 February 1967,Apollo 1
+        10d 20h 09m 03s,11 October 1968,Apollo 7
+        06d 03h 00m 42s,21 December 1968,Apollo 8
+
+    While this will throw off pd.read_csv, it can be handled with some minor
+    pre-processing.
```

Comment:
I'm not sure I agree with the decision to put this right in the documentation.  Is this truly a use case that needs to be this up front and center?  I understand that this is not a rare use-case, but I feel this is better suited in a section of its own in `io.rst` (without having to be in the docs).

Keep in mind that this example isn't just about the file-handler.  It also touches on comments, and without our docstring as large as it is, we should be relatively selective as to what examples we drop in there.

Secondly, the phrase is "throw off" is a little too colloquial IMO for the docs.  For someone who's unfamiliar with our API, it's not really clear why this data would "throw off" `read_csv`.  It would be better to explain what you are trying to do and show how `read_csv` can in part help you achieve it (with this workaround).

Reply:
@gfyoung you're right. I misunderstood the earlier requests. I'll be making the change and pushing it up soon. I'll modify the phrasing and ping you.